app/views/game/mini.scala

```diff
@@ -53,8 +53,14 @@ object mini:
       renderPlayer(pov, withRating = showRatings)
     )
 
-  def renderState(pov: Pov) =
-    dataState := s"${Fen writeBoardAndColor pov.game.situation},${pov.color.name},${~pov.game.lastMoveKeys}"
+  def renderState(pov: Pov)(using me: Option[Me]) =
+    val fen =
+      if me.exists(pov.player.userId.has) && pov.player.blindfold.nonEmpty ||
```

Comment:
`if me.exists(pov.player.isUser)`

Reply:
Probably not useful outside of blindfold mode, but empty board fen could be a constant in scalachess as well.